crates/sc-consensus-subspace/src/archiver.rs

```diff
@@ -753,6 +775,9 @@ fn finalize_block<Block, Backend, Client>(
 /// processing, which is necessary for ensuring that when the next block is imported, inherents will
 /// contain segment header of newly archived block (must happen exactly in the next block).
 ///
+/// When a block with object mappings is archived, notification ([`SubspaceLink::archived_mapping_notification_stream`])
```

Comment:
```suggestion
/// When a block with object mappings is produced, notification ([`SubspaceLink::archived_mapping_notification_stream`])
```

It is not the object mapping that we're archiving, object mapping is the result of the block archiving process.